examples/conv-opt/comparison/README.md

```diff
@@ -33,6 +33,13 @@ $ cd buddy-mlir/examples/conv-opt/comparison/
 $ python3 pytorch-conv2d.py
 ```
 
+- ONNX Runtime
+
+```
+$ cd buddy-mlir/examples/conv-opt/comparison/
+$ python3 onnxruntime-conv2d.py
```

Comment:
It seems the onnx model generation step needs to be added here.

Reply:
Added step to generate onnx models.